src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!--<h1>HTML Form</h1>-->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">   <!-- Personal information-->
+        <legend>Personal information</legend>
+        <div class="inputs">
+          <label>Surname:
```

Comment:
chekclist p2 fix in all cases
```suggestion
          <label>
          Surname:
```